### ISSUE TITLE
Captions do not change in size with ctrl++/ctrl+- (desktop browsers)

### ISSUE THREAD

**Issue by apadhye** [mozilla/vtt.js#336]
We cannot use keyboard options to increase/decrease font size.

*STR:*
- Create a video with captions.
- Play the video and enable captions.
- Use Ctrl+ or Ctrl- (Cmd+ and Cmd- if you are on a Mac) to increase/decrease browser text size.

*Expected:*
- Caption size increases or decreases along with the rest of the browser text.

*Actual:*
- Caption size remains the same.

**Comment by RickEyre** [comment]
Hey @apadhye, thanks for your feedback. I think this might be a bug that better belongs on [bugzilla](https://bugzilla.mozilla.org/). While parts of this issue most likely do touch this repo, most of it would probably be on the browser side.

**Comment by gkatsev** [comment]
We're seeing this particularly in videojs where we're using vttjs to implement captions. We weren't sure whether vttjs could do anything about it.

**Comment by RickEyre** [comment]
@gkatsev yeah, I'm unsure right now on what would be causing this. I don't really have much time to investigate either. I'd be curious to know how the browser effects the size change when hitting +/-. That would tell us a lot about if `vtt.js` could do anything about it.

`vtt.js` calculates the size of the cue boxes and font via a relative percentage of the containing box. So I would think it would pick up on it.. something is getting lost in the mix though.
